setup.py

```diff
@@ -21,10 +21,30 @@
     # For Python 2.*, install the backported subprocess
     REQUIREMENTS.append('subprocess32')
 
-TEST_REQUIREMENTS = ['coverage', 'mock', 'nose']
+TEST_REQUIREMENTS = ['coverage', 'mock', 'nose', 'pytest', 'pytest-cov']
 
 LINT_REQUIREMENTS = ['flake8']
 
+
+class ContinuousIntegrationCommand(Command):
```

Comment:
What are your plans are for class `ContinuousIntegrationCommand(setuptools.Command)`?

Reply:
With this PR, I am suggesting that we use this pattern for all libraries that does not come with any `requirements.txt` or `Pipfile`.